src/controller/CHIPDevice.h

```diff
@@ -525,6 +546,12 @@ class DLL_EXPORT Device : public Messaging::ExchangeDelegate, public SessionEsta
     PersistentStorageDelegate * mStorageDelegate = nullptr;
 
     uint8_t mCSRNonce[kOpCSRNonceLength];
+    uint8_t mAttestationNonce[kAttestationNonceLength];
```

Comment:
Note for @pan-apple, @msandstedt 

This storage shouldn't be paid by the Device object, neither should the CSR Nonce (separate issue). Otherwise, when embedded Nodes start trying to control other nodes for bindings, they need, based on current structure, Device objects. They will need to carry the cost of this nonce (and CSR nonce) due to the storage here.

Reply:
Yes, this should be removed from here. We can probably add a TODO here and address it in a separate PR.

Reply:
Agreed.  This might seem convenient because this data is associated with the Device, but `chip::Controller::DeviceCommissioner` can't be storing large pieces of data in the base Device object like this.

Reply:
added TODO